playbooks/roles/install_lbaasv2_driver/tasks/main.yml

```diff
@@ -21,6 +21,11 @@
     state: present
   when: not use_pip|bool
 
+- name: Install git
+  package:
+    name:  git-1.8.3.1-6.el7_2.1
```

Comment:
I think this should be reference via a default variable, which would be added here:

https://github.com/f5devcentral/f5-openstack-ansible/blob/liberty/playbooks/roles/configure_lbaasv2_driver/vars/main.yml

And the user can override in their playbook if they desire.